### ISSUE TITLE
reverse arguments order in `tablature` function to allow piping

### ISSUE THREAD

**Issue by sam217pa** [ezgraphs/ggguitar#5]
I would suggest exchanging the `desc` and `name` argument of the tablature function to allow something like: 

```
c(3, 2, 0, 0, 0, 3) %>% tablature('Gmaj')
```

instead of doing the less "tidyverse-idiomatic":

```
c(3, 2, 0, 0, 0, 3) %>% tablature('Gmaj', . )
```

Thanks for your nice package 😃  !

**Comment by ezgraphs** [comment]
Yeah - good point.  Adding this issue to a milestone where changes to function names / argument lists will be implemented.
